Tests/configure_and_test_integration_instances.py

```diff
@@ -959,70 +1122,134 @@ def main():
-        prints_manager.add_print_job('Start of Instance Testing ("Test" button) after the Content Update:',
-                                     print_warning, 0)
-    else:
-        prints_manager.add_print_job('No integrations to configure for the chosen tests. (Post-update)',
-                                     print_warning, 0)
+            successful_tests.add((instance_name, integration_of_instance))
+
+    return successful_tests, failed_tests
+
+
+def update_content_till_v6(build: Build):
+    threads_list = []
+    threads_prints_manager = ParallelPrintsManager(len(build.servers))
+    # For each server url we install content
+    for thread_index, server_url in enumerate(build.servers):
+        client = demisto_client.configure(base_url=server_url, username=build.username,
+                                          password=build.password, verify_ssl=False)
+        t = Thread(target=update_content_on_demisto_instance,
+                   kwargs={'client': client, 'server': server_url, 'ami_name': build.ami_env,
+                           'prints_manager': threads_prints_manager,
+                           'thread_index': thread_index})
+        threads_list.append(t)
+
+    run_threads_list(threads_list)
+
+
+def disable_instances(build: Build, all_module_instances, prints_manager):
+    # for server_url in build.servers:
+    #     client = demisto_client.configure(base_url=server_url, username=build.username, password=build.password, verify_ssl=False)
+    #     __disable_integrations_instances(client, all_module_instances, prints_manager)
+    client = demisto_client.configure(base_url=build.servers[0], username=build.username, password=build.password,
+                                      verify_ssl=False)
+    __disable_integrations_instances(client, all_module_instances, prints_manager)
     prints_manager.execute_thread_prints(0)
 
-    for instance in all_module_instances:
-        integration_of_instance = instance.get('brand', '')
-        instance_name = instance.get('name', '')
-        msg = 'Testing ("Test" button) for instance "{}" of integration "{}" .'.format(instance_name,
-                                                                                       integration_of_instance)
-        prints_manager.add_print_job(msg, print_color, 0, LOG_COLORS.GREEN)
-        prints_manager.execute_thread_prints(0)
-        # If there is a failure, __test_integration_instance will print it
-        success, _ = __test_integration_instance(testing_client, instance, prints_manager)
-        prints_manager.execute_thread_prints(0)
-        if not success:
-            postupdate_fails.add((instance_name, integration_of_instance))
+
+def create_nightly_test_pack():
+    test_pack_zip(Build.content_path, Build.test_pack_target)
+
+
+def test_files(content_path):
+    packs_root, packs, _ = next(os.walk(f'{content_path}/Packs')) # lgtm [py/unguarded-next-in-generator]
```

Comment:
You can use `os.scandir()` instead, I don't think that there is need to use `os.walk` in that case.